### PR TITLE
Don't import kiva/_version.py from setup.py

### LINKED ISSUE #254: Path resolution for _version.py in setup.py
~~The setup.py file assumes it is being run from the same directory as it is in.  Pip doesn't always do this.~~  Gives this error (eg. running in an otherwise clean virtual environment):
```
Processing ./enable-4.6.0.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/qp/t7g3hmrs7tb24jhkq6b3wn_00000gq/T/pip-c_hp5a-build/setup.py", line 225, in <module>
        write_version_py(filename='enable/_version.py')
      File "/private/var/folders/qp/t7g3hmrs7tb24jhkq6b3wn_00000gq/T/pip-c_hp5a-build/setup.py", line 103, in write_version_py
        raise ImportError("Unable to import git_revision. Try removing "
    ImportError: Unable to import git_revision. Try removing kiva/_version.py and the build directory before building.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/qp/t7g3hmrs7tb24jhkq6b3wn_00000gq/T/pip-c_hp5a-build/
```

### PR DESCRIPTION
**Pull request by jwiggins**
Fixes #254 and #253

six was already in the `__requires__` list in enable/__init__.py.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -98,11 +107,11 @@ def write_version_py(filename):
     elif os.path.exists('kiva/_version.py'):
         # must be a source distribution, use existing version file
         try:
-            from kiva._version import git_revision, full_version
-        except ImportError:
-            raise ImportError("Unable to import git_revision. Try removing "
-                              "kiva/_version.py and the build directory "
-                              "before building.")
+            git_revision, full_version = read_version_py('kiva/_version.py')
```
Comment (corranwebster) [inline setup.py]: Probably need to be careful here:

* cross-platform path construction
* need to ensure that it works if `setup.py` is not in current working dir (ie. base off of `os.dirname(__file__)`).

Comment (jwiggins) [inline setup.py]: Good point. Done.

**Comment by jwiggins** [comment]
I've built the sdist from this branch on both 2.7 and 3.5. It should be good to go, right?

**Comment by corranwebster** [comment]
Yep - sorry should have said it LGTM earlier.

**Comment by jwiggins** [comment]
Better late than never!  😄

**Pull request merged**